2016-11-25-shiny-1.md

```diff
@@ -67,15 +67,15 @@ RStudio generates a template R script called `app.R`.
 
 Notice that the name given to your app was assigned to the directory, not the app script. For your app to work, the file must remain named `app.R`.
 
-It's possible to create a Shiny app with two files called `ui.R` and `server.R`, but the same can be accomplished by using one file. This tutorial will assume you have the one file app structure.
+It's possible to create a Shiny app with two files called `ui.R` and `server.R`, but the same can be accomplished by using one file. The difference between creating an app with two files and one file is... This tutorial will assume you have the one file app structure.
 
-Local data for your app needs to be stored in a directory called `Data` inside your app directory. When importing the data for use by `app.R` use the full file path, e.g.:
+Local data for your app needs to be stored in a directory called `Data` inside your app directory. To create the directory `Data`, navigate to the `Files` tab in RStudio... When importing the data for use by `app.R` use the full file path, e.g.:
```

Comment:
I think we need to spell this out, I know it seems common sense, but better to make it very clear. A screenshot of RStudio with the Files tabs open on the working directory with "New Folder" circled might be useful.